etl/storage/storage.go

```diff
@@ -386,10 +386,39 @@ func (s *Storage) GetJetDropsWithParams(pulse models.Pulse, fromJetDropID *model
 	return jetDrops, int(total), err
 }
 
-func (s *Storage) GetJetDropByID(id models.JetDropID) (models.JetDrop, error) {
+func (s *Storage) GetJetDropByID(id models.JetDropID) (models.JetDrop, []models.JetDrop, []models.JetDrop, error) {
 	var jetDrop models.JetDrop
 	err := s.db.Model(&jetDrop).Where("pulse_number = ? AND jet_id = ?", id.PulseNumber, id.JetID).Find(&jetDrop).Error
-	return jetDrop, err
+	if err != nil {
+		return jetDrop, nil, nil, err
+	}
+	// get prev and next pulse
+	var pulse models.Pulse
+	err = s.db.Where("pulse_number = ?", id.PulseNumber).First(&pulse).Error
+	if err != nil {
+		return jetDrop, nil, nil, err
+	}
+
+	siblings := []string{jetDrop.JetID, fmt.Sprintf("%s0", jetDrop.JetID), fmt.Sprintf("%s1", jetDrop.JetID)}
```

Comment:
first question: may it be a methods of jetdropid ?
second, why special variable sz ?

Reply:
first: yes, I'll make it method
second: variable sz let us to calculate len of jetDrop.JetID once, not twice